language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
```

Comment:
Usually, I don't declare a new variable if I'm going to use it only once. So, as an example, instead of:
```python
text_splitted = text.split()
text_splitted_lower = text_splitted.lower()

for word in text_splitted_lower:
    #loop here
```
I prefer:
```python
for word in text.split().lower():
    #loop here
```
Code is shorter and more readable. But I don't know if it's good practice to do this. :)